src/environment.ts

```diff
@@ -1,4 +1,12 @@
-const BASE_URL = import.meta.env.BASE_URL;
+const removeHostPartOfUrl = (fullUrl: string) => {
+    if (import.meta.env.MODE === 'test') return fullUrl;
```

Comment:
Måtte sette fullurl for at appen som kjører i modia skal hente resten av bundlene fra riktig url, resten av appen forventer relativ-url som BASE_URL men testene krever fortsatt absolutt-url så lenge man bruker jsdom